sdcm/utils/scylla_api.py

```diff
@@ -0,0 +1,54 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2022 ScyllaDB
+
+# These classes are in preliminary phase - will require more generalization when more features will be added
+# possibly this file could be generated automatically
+
+import logging
+
+from sdcm.cluster import BaseNode
+from sdcm.remote.libssh2_client import Result
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ScyllaApiException(Exception):
+    pass
+
+
+class CompactionManagerApi:  # pylint: disable=too-few-public-methods
```

Comment:
We already have a some classes for working with the REST API via curl on remoter in `/sdcm/rest`. Let's avoid duplication and choose one (there's also a `storage_service_client` implementation there already).

Reply:
yes, I forgot about this. I adapted my logic to current solution.